Sources/GRPC/GRPCLogger.swift

```diff
@@ -71,6 +73,46 @@ internal struct GRPCLogger {
       line: line
     )
   }
+
+  internal func notice(
+    _ message: @autoclosure () -> Logger.Message,
+    metadata: @autoclosure () -> Logger.Metadata? = nil,
+    file: String = #file,
+    function: String = #function,
+    line: UInt = #line
+  ) {
+    self.logger.notice(
+      message(),
+      metadata: metadata(),
+      source: "GRPC",
```

Comment:
Not `gRPC`? ;)

Reply:
Logging source is usually the module name.